src/components/FileInput.js

```diff
@@ -68,6 +70,19 @@ class FileInput extends Component {
       this.input.click();
     }
   }
+
+  getImageThumbnail(file = { mimeType: '' }) {
+    const reader = new FileReader();
+    if (file) {
+      if (file.mimeType.match('image.*')) {
+        reader.readAsDataURL(file);
+      } else {
+        this.setState({ image: '' });
+      }
+    }
+    reader.onload = e => this.setState({ image: e.target.result });
```

Comment:
Change the name of variable `e` to something more descriptive, for example: `event`